modules/virt-binding-devices-vfio-driver.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices-vfio-driver_{context}"]
+= Binding a PCI device to the VFIO driver
+
+To bind a PCI device to the VFIO driver, find the values for `vendor-ID` and `device-ID` and add them to the `/etc/modprobe.d/vfio.conf` file. Then, configure the `MachineConfig` object file for the VFIO driver and load the new driver.
+
+.Prerequisites
+. The Intel or AMD hardware must be supported by the CPU and enabled in the BIOS settings.
```

Comment:
This needs to be updated, probably we say,  `Ensure that ( IOMMU ) AMD-Vi/Intel VT-d is supported by the CPU and enabled in the BIOS settings. `

I added some notes, which the user may find helpful, as I was not entirely sure about IOMMU.

**NOTE:**
1)  IOMMU is a generic name for Intel VT-d and AMD-Vi.
2) Using IOMMU opens to features like PCI passthrough.
3) VT-d stands for Intel Virtualization Technology for Directed I/O and should not be confused with VT-x Intel Virtualization Technology.

@vladikr , What do you think ?

Reply:
On second thought, I feel this **Prerequisites** line in this current section should be moved to the "Adding kernel arguments to enable IOMMU for the CPU" Section.
And in here that is "Binding a PCI device to the VFIO driver"  Section.
**Prerequisites**
Configuring "Adding kernel arguments to enable IOMMU for the CPU"  is completed.